auth-web/src/components/auth/manage-business/AddBusinessForm.vue

```diff
@@ -1,20 +1,27 @@
 <template>
-  <div class="add-business-form">
+  <div id="add-business-form">
+    <HelpDialog
+      :helpDialogBlurb="helpDialogBlurb"
+      ref="helpDialog"
+    />
+
     <v-form ref="addBusinessForm" lazy-validation>
-      <fieldset>
-        <legend hidden>Incorporation Number and Password or Passcode</legend>
```

Comment:
Fieldset and legend can be useful for accessibility. In this case, I think it's cleaner to just attach an `aria-label` property to the inputs.